api/controllers/task.ts

```diff
@@ -0,0 +1,110 @@
+import { Request, Response } from 'express';
+import prisma from './db_service';
+
+async function getAllTasks(req: Request, res: Response) {
+  const { query } = req.query;
+  console.log("query",query);
+  const tasks = await prisma.tasks.findMany();
+  if(tasks?.length > 0) {
+    res.status(200).json(tasks);
+  } else {
+    console.log("getAllTasks",tasks)
+    res.sendStatus(404);
+}
+}
+
+async function getTask( req: Request, res: Response){
+  const { task_id } = req.params;
+  console.log("getTask")
+
+  try{
+  const task= await prisma.tasks.findUnique({
+    where: {
+      id: parseInt(task_id),
+    },
+    select:{
+      id: true,
+      title:true,
+      description:true,
+      due_date:true,
+      created_at:true,
+      task_list_id:true,
+      task_status_id:true
```

Comment:
Suggestion:
- If you started with camelCase I will stick with camelCase everywhere instead of adding snake_case in json.